### PR TITLE
[SDK] - Add ability to view and update owner of a contract

### PR DESCRIPTION
**Pull request by aeither**
Issue https://github.com/thirdweb-dev/js/issues/81

### CODE REVIEW

File: packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts [hunk 1]
```diff
@@ -225,6 +228,19 @@ export class EditionDrop extends StandardErc1155<DropERC1155> {
...
+   * Get current owner of the contract
+   *
+   * @example
+   * ```javascript
+   * const owner = await contract.getOwner();
+   * ```
+   * @returns The owner address.
+   */
+  public async getOwner(): Promise<string> {
+    return this.owner.get();
```
Comment (joaquim-verges) [inline packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts:241]: the pattern we've been doing is directly accessing these common functions using the namespace.

`owner` is public so ppl can call `contract.owner.get()` and `contract.owner.set()`

no need to re-expose these at the top level

File: packages/sdk/src/core/classes/contract-owner.ts [hunk 2]
```diff
@@ -22,14 +22,14 @@ export class ContractOwner<TContract extends Ownable>
...
    * @returns the owner address
    */
-  public async get(): Promise<string> {
+   public async get(): Promise<string> {
     return this.contractWrapper.readContract.owner();
   }
 
   /**
-   * set tje new owner of the contract
+   * set the new owner of the contract
```
Comment (joaquim-verges) [inline packages/sdk/src/core/classes/contract-owner.ts:30]: 🙏

File: packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts (hunk 1 above)
Comment (aeither) [reply packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts:241]: I deployed all the pre-built contracts with the dashboard to check in the explorer section to make sure it contains `setOwner` and `owner`.

- Edition
- Edition Drop
- NO - Marketplace
- Multiwrap
- NFT Collection
- NFT Drop
- Pack
- Signature Drop
- NO - Split
- NO - Token
- NO - Token Drop
- NO - Vote

Few contracts doesn't have owner so I implemented only on those that have.
I re-exposed at the top level as requested in the goals in the issue #81

File: packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts [hunk 3]
```diff
@@ -225,6 +228,19 @@ export class EditionDrop extends StandardErc1155<DropERC1155> {
...
+  /**
+   * Get current owner of the contract
+   *
+   * @example
+   * ```javascript
+   * const owner = await contract.getOwner();
+   * ```
+   * @returns The owner address.
+   */
+  public async getOwner(): Promise<string> {
```
Comment (joaquim-verges) [inline packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts:240]: you don't need these top level functions. Let's just expose it via the namespace: `contract.owner.get()` directly

File: .changeset/eleven-walls-sleep.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/sdk": minor
```
Comment (joaquim-verges) [inline .changeset/eleven-walls-sleep.md:2]: Can you make this a patch version instead of minor?

```suggestion
"@thirdweb-dev/sdk": patch
```

**Review by joaquim-verges — APPROVED** [review]
thank you for your contribution @aeither ! merging this. will be available in the next release

**Pull request merged**